llvm/lib/Target/RISCV/RISCVInstrInfo.cpp

```diff
@@ -1206,6 +1206,15 @@ bool RISCVInstrInfo::isPtrAddInstr(const MachineInstr &I,
   }
 }
 
+unsigned int RISCVInstrInfo::getCheriAddressSubregIdx(llvm::MVT CapTy) const {
```

Comment:
Don't need the llvm::?

Reply:
Oh yeah that was IDE autocomplete being a bit silly. Will fix.